### PR TITLE
Add network alias for DNS use.

### PR DESCRIPTION
**Pull request by andreyod**

### CODE REVIEW

File: pkg/multiTenancyPlugins/naming/nameScoping.go [hunk 1]
```diff
@@ -59,6 +60,9 @@ func (nameScoping *DefaultNameScopingImpl) Handle(command utils.CommandEnum, clu
 			newQuery = strings.Replace(r.RequestURI, r.URL.Query().Get("name"), r.URL.Query().Get("name")+r.Header.Get(headers.AuthZTenantIdHeaderName), 1)
 			config.Config.Labels[headers.OriginalNameLabel] = r.URL.Query().Get("name")
 		}
+		// Replace network reference with network ID, force Network-scoped alias for DNS use.
```
Comment (ezrasilvera) [inline pkg/multiTenancyPlugins/naming/nameScoping.go:62]: Why not moving the two lines bellow to inside the if above ?

Comment (andreyod) [inline pkg/multiTenancyPlugins/naming/nameScoping.go:62]: Use case: user supplied --net and --net-alias without --name. We didn't support this use case until now.

Comment (ezrasilvera) [inline pkg/multiTenancyPlugins/naming/nameScoping.go:62]: But this is not the case here - you get teh --name parameter  into the config. Thi swill be non empty only in the if , is it ?
Also we do support somone doing --net-alias as this goes directly to the engine

Comment (andreyod) [inline pkg/multiTenancyPlugins/naming/nameScoping.go:62]: Goes to the engine with network name without the prefix and fail. Additional name scoping is needed when --net-alias is provided.

**Pull request merged**